README.md

```diff
@@ -0,0 +1,170 @@
+typelevel website
```

Comment:
We added a lot of instructions to the , and continue to do so.  Some of them are no longer relevant, but I don't want to lose it.